boa/src/builtins/date/tests.rs

```diff
@@ -1,10 +1,12 @@
+#![allow(clippy::zero_prefixed_literal)]
```

Comment:
Why do we need this?

Reply:
This was generating `145` errors in my IDE, Many `Date` tests are as follows `IsDate(2020, 07, 02)` the include a `0` prefix in months and days.